.github/workflows/python-bare-executions.yml

```diff
@@ -33,10 +33,13 @@ jobs:
           ref: ${{ inputs.ref }}
           persist-credentials: false
           submodules: "recursive"
-      - name: Set up Python ${{ env.PYTHON_VERSION }}
+          fetch-depth: 2
+      - name: Set Python version vars
+        uses: ./.github/actions/build_info
+      - name: Set up Python ${{ env.PYTHON_MAX_VERSION }}
         uses: actions/setup-python@v5
```

Comment:
Wha is this here?

Reply:
Oh, I thought I added a comment on that as well.. but cannot find it. The bare execution test doesn't work correctly right now with our constraint process. And since I updated playwright, I need this test to not use constraints when I use `dev:upgrade-dependencies`. These changes make this possible.